frontend/src/components/bucket/BucketPermission.vue

```diff
@@ -70,141 +76,150 @@ onBeforeMount(async () => {
+            <Checkbox
+              v-model="data.create"
+              input-id="create"
+              aria-label="upload"
+              :binary="true"
+              @update:model-value="(value: boolean) => updateBucketPermission(value, data.userId, Permissions.CREATE)"
+            />
+          </template>
+        </Column>
+        <Column
+          header="Read"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.read"
+              input-id="read"
+              aria-label="Read"
+              :binary="true"
+              @update:model-value="(value: boolean) => updateBucketPermission(value, data.userId, Permissions.READ)"
+            />
+          </template>
+        </Column>
+        <Column
+          header="Update"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.update"
+              input-id="update"
+              aria-label="Update"
+              :binary="true"
+              @update:model-value="(value: boolean) => updateBucketPermission(value, data.userId, Permissions.UPDATE)"
+            />
+          </template>
+        </Column>
+        <Column
+          header="Delete"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.delete"
+              input-id="delete"
+              aria-label="delete"
+              :binary="true"
+              @update:model-value="(value: boolean) => updateBucketPermission(value, data.userId, Permissions.DELETE)"
+            />
+          </template>
+        </Column>
+        <Column
+          header="Manage"
+          header-class="header-center"
+          body-class="content-center"
+        >
+          <template #body="{ data }">
+            <Checkbox
+              v-model="data.manage"
+              input-id="manage"
+              aria-label="manage"
+              :binary="true"
+              :disabled="!data.elevatedRights"
+              @update:model-value="(value: boolean) => updateBucketPermission(value, data.userId, Permissions.MANAGE)"
+            />
+          </template>
+        </Column>
+        <Column header="Remove">
+          <template #body="{ data }">
+            <Button
+              class="p-button-lg p-button-text"
+              severity="danger"
+              aria-label="remove"
+              @click="removeBucketUser(data.userId)"
+            >
+              <font-awesome-icon icon="fa-solid fa-user-xmark" />
+            </Button>
+          </template>
+        </Column>
+      </DataTable>
+    </TabPanel>
+    <TabPanel header="Bulk add/remove">
+      <BulkPermission
+        resource-type="bucket"
+        :resource="bucket"
```

Comment:
despite the diff looking big, the only change to this file is putting the content into two tabs, and passing the `bucket` to the BulkPermission component.